doc/release-notes/release-7.sgml.in

```diff
@@ -205,7 +205,10 @@ This section gives an account of those changes in three categories:
 <sect1>Changes to existing options<label id="modifiedoptions">
 <p>
 <descrip>
-	<p>No changed options in this version.
+	<tag>--with-ldap</tag>
+	<p>Removed special support for outdated Mozilla LDAP SDK.
+	   <em>--with-ldap=/path/to/sdk CPPFLAGS="-I/path/to/sdk/mozldap"</em>
+	   can be used if needed.
```

Comment:
Implication is correct: Squid default builds **will not work** with Mozilla LDAP SDK after this patch is applied.

The custom path and CPPFLAGS are required for that LDAP SDK to be working at all.

Reply:
> Implication is correct: Squid default builds will not work with Mozilla LDAP SDK after this patch is applied.

The above implication is not the false implication I recommend avoiding in my change request.

Reply:
Then why did you use those words?  "false implication that Squid no longer works with Mozilla LDAP SDK"

Reply:
> Then why did you use those words? "false implication that Squid no longer works with Mozilla LDAP SDK"

I used the words I used because I believed they accurately described the problem. I still believe that they do.

There is obviously a difference between my description and the description in your [response](https://github.com/squid-cache/squid/pull/1710/files#r1512499309). The two phrases do not describe the same thing. The description in your response is about whether "default builds" work with Mozilla LDAP SDK. The concept of "support" goes beyond default builds, of course -- Squid may support X even though Squid default build does not support X. My description is about that general "support" concept.